pkg/gandidns/README.md

```diff
@@ -0,0 +1,136 @@
+# GableBee
+
+> Gandi Api BlockLayer Bot
+
+This is a cuelang module for BlockLayer meant to manipulate DNS zones through Gandi LiveDNS.
+
+Additionally, it provides agnostic base cue building blocks making it easy to add support for other DNS providers.
+
+## TL;DR
+
+Create your `main.cue`:
+
+```
+package foo
+
+import (    
+  "blocklayer.dev/gandidns"
+)
+
+yourDomain: gandidns.Zone & {
+  // Your super domain name
+  domain: "yourdomainname.tld"
+  // By default, this will REPLACE your existing zone with this one - alternatively, set this to "merge"
+  mode: "replace"
+
+  provider: {
+    settings: {
+      // Pass your Gandi API key
+      apikey: "gandikey"
+    }
+  }
+
+  // Add as many DNS records as you want
+  records: [
+    // Here is an A record for @ to point to 1.1.1.1
+    gandidns.Records.A & {
+      name: "@"
+      value: "1.1.1.1"
+    }
+  ]
+}
+```
+
+## Moar
+
+Check the `EXAMPLE.md` file for a more involved example.
+
+## Requirements
+
+ * BlockLayer alpha 3
+ * a domain name on `gandi.net` with livedns enabled
+ * a working Gandi API key for your account
+
+## API
+
+### Zone
+
+```
+Zone:: {
+  // An optional ttl to apply to all records in the zone by default (can be overriden per record)
+  ttl: uint32 | * 300
+
+  // The domain name to which this zone will be attached
+  domain: string
+
+  // By default ("replace"), the entire zone is being replaced by the provided records.
+  // In the "merge" mode, only the records that have the same "type" and "name" will be replaced,
+  // and the others will be kept.
+  // Note that NS records are ALWAYS left untouched and you can't specify them right now
+  mode: =~ "^(replace|merge)$" | * "replace"
+
+  provider: {
+    settings: {
+      // Your Gandi API key - see Gandi documentation
```

Comment:
Nit: bonus points for an http link to the Gandi docs :)

Reply:
And fixed!